client/Lykke.Service.PaymentSystem.Client/Extensions/PaymentSystemExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Service.PaymentSystem.Client.AutorestClient;
+using Lykke.Service.PaymentSystem.Client.AutorestClient.Models;
+using Lykke.Service.PaymentSystem.Client.Extensions;
+
+namespace Lykke.Service.PaymentSystem.Client
+{
+    /// <summary>
+    /// PaymentSystem extensions
+    /// </summary>
+    public static class PaymentSystemExtensions
+    {
+        /// <summary>
+        /// GetInfo as object
+        /// </summary>
+        /// <param name="src"></param>
+        /// <param name="expectedType"></param>
+        /// <param name="throwExeption"></param>
+        /// <returns></returns>
+        public static object GetInfo(this PaymentTransactionResponse src, Type expectedType = null, bool throwExeption = false)
```

Comment:
why do we need "object" as info in any client?

Reply:
Honestly copied extension. :)
I altered and left one generic method.